pkgs/servers/nextcloud/packages/generate.sh

```diff
@@ -1,14 +1,15 @@
 #!/usr/bin/env nix-shell
-#! nix-shell -I nixpkgs=../../../.. -i bash -p nc4nix
+#!nix-shell nc4nix-shell.nix -i bash
 
 set -e
 set -u
 set -o pipefail
 set -x
 
-export NEXTCLOUD_VERSIONS=$(nix-instantiate --eval -E 'import ./nc-versions.nix {}' -A e)
+echo "versions: $NEXTCLOUD_VERSIONS"
 
-APPS=`cat nextcloud-apps.json | jq -r '.[]' | sed -z 's/\n/,/g;s/,$/\n/'`
+APPS=$(jq -r '.[]' "$SCRIPT_FOLDER"/nextcloud-apps.json | sed -z 's/\n/,/g;s/,$/\n/')
 
-nc4nix -apps $APPS
-rm *.log
+# nc4nix needs NEXTCLOUD_VERSIONS
+nc4nix -apps "$APPS"
+rm ./*.log
```

Comment:
```suggestion
rm -f *.log
```
don't fail if somehow we didn't generate a log